tokio-executor/Cargo.toml

```
@@ -23,8 +23,8 @@ categories = ["concurrency", "asynchronous"]
 publish = false
 
 [dependencies]
-# crossbeam-utils = "0.6.2"
 crossbeam-utils = { git = "https://github.com/stjepang/crossbeam", branch = "raw-parker" }
 
 [dev-dependencies]
-# tokio = { version = "0.2.0", path = "../tokio" }
+futures-core-preview = "0.3.0-alpha.17"
```

```suggestion
futures-core-preview = "=0.3.0-alpha.17"
```

I think we want to do this cuz alpha 18 could be breaking?

---

Probably.